staging/src/k8s.io/csi-api/pkg/crd/manifests/csidriver.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: csidrivers.csi.storage.k8s.io
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  group: csi.storage.k8s.io
+  names:
+    kind: CSIDriver
+    plural: csidrivers
+  scope: Cluster
+  validation:
```

Comment:
Shall csiDrivers participate the spec/status convention? See https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go#L54.

Reply:
I don't think I understand your question. We have a spec for the object, and will consider adding a status in the future if needed.